install pytest dependencies based on missing pytest config ini keywords

See run [9962badc-2024-492c-8eb1-553910b00295](https://janitor.debian.net/cupboard/pkg/pytest-services-upstream/9962badc-2024-492c-8eb1-553910b00295/).

```
cachedir: .pytest_cache
rootdir: /&lt;&lt;PKGBUILDDIR&gt;&gt;, configfile: tox.ini
collecting ... collected 0 items

=============================== warnings summary ===============================
/usr/lib/python3/dist-packages/_pytest/config/__init__.py:1148
  /usr/lib/python3/dist-packages/_pytest/config/__init__.py:1148: PytestConfigWarning: Unknown config ini key: pep8maxlinelength
  
    self._warn_or_fail_if_strict("Unknown config ini key: {}\n".format(key))

-- Docs: https://docs.pytest.org/en/stable/warnings.html
============================== 1 warning in 0.01s ==============================
E: pybuild pybuild:353: test: plugin distutils failed with: exit code=5: cd /&lt;&lt;PKGBUILDDIR&gt;&gt;/.pybuild/cpython3_3.9/build; python3.9 -m pytest 
dh_auto_test: error: pybuild --test --test-pytest -i python{version} -p 3.9 returned exit code 13
make: *** [debian/rules:4: binary] Error 25
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
```

pep8maxlinelength => python3-pytest-pep8

--- a/src/dependencies/pytest.rs
+++ b/src/dependencies/pytest.rs
@@ -59,6 +59,7 @@ fn map_pytest_arguments_to_plugin(args: &[&str]) -> Option<&'static str> {
 fn map_pytest_config_option_to_plugin(name: &str) -> Option<&'static str> {
     match name {
         "asyncio_mode" => Some("asyncio"),
+        "pep8ignore" | "pep8maxlinelength" | "pep8select" => Some("pep8"),
         n => {
             log::warn!("Unknown pytest config option {}", n);
             None
@@ -232,6 +233,10 @@ mod tests {
         assert_eq!(
             map_pytest_config_option_to_plugin("asyncio_mode"),
             Some("asyncio")
+        );
+        assert_eq!(
+            map_pytest_config_option_to_plugin("pep8maxlinelength"),
+            Some("pep8")
         );
         assert_eq!(map_pytest_config_option_to_plugin("unknown_option"), None);
     }

